StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/SavedPaymentMethodCollectionView.swift

```diff
@@ -208,15 +245,20 @@ extension SavedPaymentMethodCollectionView {
             }
         }
 
-        // MARK: - Internal Methods
+        private var labelBottomConstraint: NSLayoutConstraint?
+        private var labelHeightConstraint: NSLayoutConstraint?
```

Comment:
Seems like these do not need to be optional.  We can initialize them in the init(), and then activate/deactivate when needed.